### DISCUSSION TITLE
Walkthrough for Syncing Onedrive Business

### DISCUSSION THREAD

**Discussion by Lycistus** [abraunegg/onedrive#1781] [Q&A] [answered]
I was hoping someone may have a walkthrough for syncing Onedrive Business accounts. It looks like the link on the post below is dead. I have access to the Microsoft side of things so if I need to give permissions somewhere I can do so.  I just need to know where to do it.

https://www.linuxuprising.com/2020/02/how-to-keep-onedrive-in-sync-with.html

**Comment by abraunegg** [comment]
Have you reviewed the documents: https://github.com/abraunegg/onedrive/tree/master/docs ?

**Reply by Lycistus** [reply]
This is my bad. I should have provided a lot more info.  I didn't build or check the package myself as I am not the one installing it on a Linux machine and I am not sure where in the process the individual has his problem.  This person just said they needed permission to have the package work.  I should have checked to see if the package is correct and he followed the installation instructions, he set this up on an Ubuntu machine.   

Assuming he has set it up correctly should he be receiving a message stating:



           Need admin approval 
           OneDrive Client For Linux
           unverified
           
           OneDrive Client for Linux needs permission to access resources in your organization 
           that only an admin can grant.  Please ask an admin to grant permission to this app 
           before you can use it.

Who is he supposed to ask?  Do I need to grant the app permission in Azure?  I am sure that the individual wanting to use your software is an enthusiast and could probably work out what he has to do as long as there isn't something I need to do on my end with the Microsoft side of things.

**Comment by abraunegg** [comment, marked as answer]
@Lycistus 
> Who is he supposed to ask? 

If the application needs authorization | permission, it needs to be granted by an Azure AD Admin

> Do I need to grant the app permission in Azure?

Most likely - yes.

**Reply by kvn95ss** [reply]
I have similar issue. As the message reads 'Need admin approval / unverified ', our institute IT department has refused access. I am not getting the name 'OneDrive Client for Linux' to show up in the error, my guess is that would cure their uneasiness. Any way to prove to them this is not malicious?

**Reply by abraunegg** [reply]
@kvn95ss 
> Any way to prove to them this is not malicious?

Point them at this github repository ?
